src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/net451/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,33 @@
+<!--
+Copyright (c) .NET Foundation. All rights reserved.
+Licensed under the Apache License, Version 2.0.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MvcRazorContentRoot Condition="'$(MvcRazorContentRoot)'==''">$(MSBuildProjectDirectory)</MvcRazorContentRoot>
+    <MvcRazorOutputPath Condition="'$(MvcRazorOutputPath)'==''">$(OutputPath)</MvcRazorOutputPath>
+    <MyGeneratedDll>$(AssemblyName).PrecompiledViews.dll</MyGeneratedDll>
+    <MyGeneratedDllPath>$(OutputPath)$(MyGeneratedDll)</MyGeneratedDllPath>
+    <BuildDependsOn>$(BuildDependsOn);MvcRazorPrecompile</BuildDependsOn>
```

Comment:
...originally I thought this was the best way to go, but had a chat with MSBuild guys recently. 'AfterTargets' is the preferred way to do this. MSBuild is actually trying to get away from these 'dependson' chains.